application.go

```diff
@@ -80,7 +80,7 @@ func init() {
 	isGoTest = strings.Contains(strings.ToLower(os.Args[0]), "test")
 
 	// Get the root resource directory.
-	if globals.ResourceDir, err = gz.ReadEnvVar("IGN_FUEL_RESOURCE_DIR"); err != nil {
+	if globals.ResourceDir, err = gz.ReadEnvVar("IGN_FUEL_RESOURCE_DIR"); err != nil && !isGoTest {
```

Comment:
`globals.ResourceDir` defaults to a different value when running tests, this value is set in a different place, making the `IGN_FUEL_RESOURCE_DIR` useless, although still required. This change allows you to avoid setting the `IGN_FUEL_RESOURCE_DIR` when testing, since it will be overwritten.